cuddly-chameleons/cuddly_chameleons/retro_news/models.py

```diff
@@ -0,0 +1,32 @@
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+
+
+class CustomUser(AbstractUser):
+    """Extend default user for case when we need to add something to this."""
+
+    pass
```

Comment:
Did you know that this `pass` doesn't need to be here? I don't really mind that it is there, but it is possible to have a class which contains nothing as long as it has a docstring. The docstring counts as sufficient content.